README.md

```diff
@@ -94,43 +94,43 @@ Arguments:
 *   MBID [(MusicBrainz identifier)](https://wiki.musicbrainz.org/MusicBrainz_Identifier)
 
 ```js
-const artist = await mbApi.lookupEntity('artist', {query: 'ab2528d9-719f-4261-8098-21849222a0f2'});
+const artist = await mbApi.lookupEntity('artist', 'ab2528d9-719f-4261-8098-21849222a0f2');
 ```
 
 ### Lookup area
 
 ```js
-const area = await mbApi.lookupArea({query: 'ab2528d9-719f-4261-8098-21849222a0f2'});
+const area = await mbApi.lookupArea('ab2528d9-719f-4261-8098-21849222a0f2');
 ```
 
 ### Lookup artist
 
 Lookup an `artist` and include their `releases`, `release-groups` and `aliases`
 
 ```js
-const artist = await mbApi.lookupArtist({query: 'ab2528d9-719f-4261-8098-21849222a0f2'});
+const artist = await mbApi.lookupArtist('ab2528d9-719f-4261-8098-21849222a0f2');
 ```
 
 ### Lookup instrument
 
 Lookup an instrument
 
 ```js
-const artist = await mbApi.lookupInstrument({query: 'b3eac5f9-7859-4416-ac39-7154e2e8d348'});
+const artist = await mbApi.lookupInstrument('b3eac5f9-7859-4416-ac39-7154e2e8d348');
```

Comment:
just one more thing i noticed is that this says artist in the var name instead of instrument fyi

Reply:
ty
found some more places that needed an update